### PR TITLE
[8.0] Syntax upgrade for MonitoringSystem

### PR DESCRIPTION
**Pull request by rupozzi**

### CODE REVIEW

File: src/DIRAC/MonitoringSystem/Client/MonitoringClient.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class MonitoringClient(Client):
     def __init__(self, **kwargs):
         """Simple constructor"""
 
-        super(MonitoringClient, self).__init__(**kwargs)
+        super().__init__(**kwargs)
```
Comment (fstagni) [inline src/DIRAC/MonitoringSystem/Client/MonitoringClient.py:20]: This, for a reason I had no time to investigate, does not work. Restore it as before.

**Comment by fstagni** [comment]
You can, if you have time, convert to f-strings.

**Comment by chrisburr** [comment]
@rupozzi @atsareg  I think this one can be included today?

**Comment by rupozzi** [comment]
> @rupozzi @atsareg I think this one can be included today?

Sure, go for it.

**Review by chrisburr — APPROVED** [review]

**Pull request merged**